app/controllers/submission_controller.rb

```diff
@@ -1,10 +1,11 @@
 class SubmissionController < ApplicationController
   def create
-    @submission = Submission.new(
-      submission_params(params).merge(status: Submission::STATUS[:queued])
+    @submission = Submission.create(
+      submission_params.merge(
+        status: Submission::STATUS[:queued],
+        payload: payload
+      )
     )
-    @submission.save!
```

Comment:
the bang operator disappeared. so `id` of submission could be `nil`. is that desired?

Reply:
i used the create method so it will explode on the line before insted

Reply:
fixed